presidio-analyzer/analyzer/pattern_recognizer.py

```diff
@@ -12,8 +14,14 @@
 
 class PatternRecognizer(LocalRecognizer):
 
-    def __init__(self, supported_entity, name=None, supported_language='en',
-                 patterns=None,
+    CONTEXT_SIMILARITY_THRESHOLD = 0.65
+    CONTEXT_SIMILARITY_FACTOR = 0.35
+    MIN_SCORE_WITH_CONTEXT_SIMILARITY = 0.6
+    CONTEXT_PREFIX_COUNT = 5
+    CONTEXT_SUFFIX_COUNT = 0
+
```

Comment:
These should be in the entity_recognizer as there could be cases where we use context for non-pattern recognizers